plugins/isso.lisp

```diff
@@ -0,0 +1,20 @@
+(defpackage :coleslaw-isso
+  (:use :cl)
+  (:export #:enable)
+  (:import-from :coleslaw #:add-injection
+                          #:post))
+
+(in-package :coleslaw-isso)
+
+(defvar *isso-header*
+  "<div class=\"comments\">
+      <section id=\"isso-thread\"></section>
+      <script data-isso=\"~a/\"
+              src=\"~a/js/embed.min.js\"></script>
+    </div>")
+
+(defun enable (&key isso-url)
+  (flet ((inject-p (x)
+           (when (typep x 'post)
+             (format nil *isso-header* isso-url))))
```

Comment:
There is actually a bug here as your format string `*isso-header*` takes two arguments but you are only passing it one argument `isso-url`. I believe you should be calling `(format nil *isso-header* isso-url isso-url)`.